signer/tests/integration/emily.rs

```diff
@@ -372,6 +406,26 @@ async fn deposit_e2e() {
                 .expect_get_account()
                 .once()
                 .returning(|_| Box::pin(async move { Err(Error::InvalidStacksResponse("mock")) }));
+
+            client.expect_get_sortition_info().returning(move |_| {
+                let response = Ok(SortitionInfo {
```

Comment:
suggestion: not sure if this might be a little cleaner (smaller) if you faked the `SortitionInfo` as a `mut` and set `burn_block_hash` (and `was_sortition`?) separately? I feel like the newing-up/defaults detract a little bit from the readability of what's important.

Reply:
We don't have a faker for it (`SortitionInfo` comes from stackslib), but point taken: moved the dummy value definition outside to declutter this in [6e03268](https://github.com/stacks-network/sbtc/pull/730/commits/6e032680bcc3410e25de55698464945fdc7af486)

Reply:
Ah gotcha, well your solution solves it anyway and it's much easier to read! thx :)